code/WorkInProgress/Cael_Aislinn/Rust/gyrotron_controller.dm

```diff
@@ -1,13 +1,25 @@
-#define RUST_GYROTRON_RANGE 25
-
 /obj/machinery/computer/rust_gyrotron_controller
 	name = "Gyrotron Remote Controller"
 	icon_state = "engine"
 	circuit = /obj/item/weapon/circuitboard/rust_gyrotron_control
 	light_color = LIGHT_COLOR_BLUE
 
+	var/id_tag // Required for multitool buffering
```

Comment:
Uh, is it actually? If it is then you should fix that instead.

Reply:
Indeed, I don't really want to do it right now, though.